### PR TITLE
Enable authorized scope selection dropdown after the available scopes are populated

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

https://github.com/wso2/identity-apps/assets/41533942/155cce7d-0e17-4f75-8181-32c9206757fe

### Related Issues
- https://github.com/wso2/product-is/issues/19088

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx [hunk 1]
```diff
@@ -518,9 +502,90 @@ export const AuthorizeAPIResource: FunctionComponent<AuthorizeAPIResourcePropsIn
...
+                                                                    }
+                                                                </Typography>
+
+
+                                                                {
+                                                                    isScopeSelectDropdownReady
+                                                                        ? (
+                                                                            <Text
+                                                                                size={ 12 }
+                                                                                className="mb-1"
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx:526]: refrain from using these css classes.

File: apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx [hunk 2]
```diff
@@ -518,9 +502,90 @@ export const AuthorizeAPIResource: FunctionComponent<AuthorizeAPIResourcePropsIn
...
+                                                                            >
+                                                                                { t("common:loading") + "..." }
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx:528]:
```suggestion
                                                                                { t("common:loading") }
```

File: apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx (hunk 1 above)
Comment (Achintha444) [reply apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx:526]: Check other places

File: apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx (hunk 2 above)
Comment (pavinduLakshan) [reply apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx]: Fixed with a632512

File: apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx (hunk 1 above)
Comment (pavinduLakshan) [reply apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx]: Fixed with a1a9d33

**Review by Achintha444 — APPROVED** [review]

**Pull request merged**